### PR TITLE
rust: add a keybinding for `jump to definition`

### PR DESCRIPTION
**Pull request by isphinx**

### CODE REVIEW

File: layers/+lang/rust/packages.el [hunk 1]
```diff
@@ -40,12 +40,15 @@
...
+      (spacemacs/declare-prefix-for-mode 'rust-mode "mc" "cargo")
+      (spacemacs/declare-prefix-for-mode 'rust-mode "mg" "goto")
       (spacemacs/set-leader-keys-for-major-mode 'rust-mode
         "cc" 'spacemacs/rust-cargo-build
         "ct" 'spacemacs/rust-cargo-test
         "cd" 'spacemacs/rust-cargo-doc
         "cx" 'spacemacs/rust-cargo-run
-        "cC" 'spacemacs/rust-cargo-clean))))
+        "cC" 'spacemacs/rust-cargo-clean
+        "gg" 'racer-find-definition))))
```
Comment (robbyoconnor) [inline layers/+lang/rust/packages.el]: Shouldn't this be `g`?

**Comment by robbyoconnor** [comment]
See line comments -- this isn't entirely correct I don't think...did you test it ;)

**Comment by robbyoconnor** [comment]
Disregard any previous comments, :+1:

**Comment by syl20bnr** [comment]
Excellent! Thank you !
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**